CrossAppLogin/Front/src/main/kotlin/com/infomaniak/core/crossapplogin/front/components/CrossLoginBottomContent.kt

```diff
@@ -318,8 +348,11 @@ private fun ButtonExpanded(
 }
 
 object CrossLoginBottomContentDefaults {
-    val buttonShape = RoundedCornerShape(16.dp)
-    val primaryButtonHeight = 56.dp
+    val primaryButtonType = object : ButtonStyle {
+        override val height = Dimens.buttonHeight
+        override val shape = RoundedCornerShape(Dimens.largeCornerRadius)
+    }
+    val nextButtonShape = CircleShape
```

Comment:
We should keep the old style as the default nextButtonShape and keep overriding it to CircleShape in kDrive which is the odd one out here